### PR TITLE
#163154901  create header component 

### PR DESCRIPTION
**Pull request by el-Joft**
#### What does this PR do?
-This PR creates the Header Component
#### Description of Task to be completed?
- The Header Component consists of the logo, nav-items, signup and login buttons, and the search
fields 
#### How should this be manually tested?
- npm test
#### What are the relevant pivotal tracker stories?
- [#163154901](https://www.pivotaltracker.com/story/show/163154901)

#### Screenshots
- The UI design 
 
image
image

- The Finished design
image
- Finished 
image

### CODE REVIEW

File: src/component/UI/Header/Header.scss [hunk 1]
```diff
@@ -0,0 +1,107 @@
+$cardo: 'Cardo', serif;
+$dropdown-content: #f1f1f1;
+$dropdown-content-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+$primary-color: #606E86;
+$secondary-color : #ffff;
+$reem-kufi: 'Reem Kufi', sans-serif;
```
Comment (pitaz) [inline src/component/UI/Header/Header.scss]: Hi @el-Joft,
Please make use of the global scss variables for consistency. For instance `$reem-kufi` has already been defined. Thanks

**Comment by azupatrick0** [comment]
Nice work @el-Joft

**Comment by sudhons** [comment]
Please what are these `.idea/.../...` files?

File: .babelrc [hunk 2]
```diff
@@ -17,6 +17,8 @@
 },
 "plugins": [
   "@babel/plugin-proposal-class-properties",
-  "@babel/plugin-syntax-dynamic-import"
+  // "@babel/plugin-proposal-object-rest-spread",
```
Comment (henperi) [inline .babelrc]: Please take off the comments on this file

File: .idea/workspace.xml [hunk 3]
```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (henperi) [inline .idea/workspace.xml]: Please take off the idea folder.

File: src/component/Pages/HomePage.jsx [hunk 4]
```diff
@@ -98,3 +98,12 @@ const HomePage = () => (
 );
 
 export default HomePage;
+
```
Comment (henperi) [inline src/component/Pages/HomePage.jsx:101]: Please take these comments off

File: .babelrc (hunk 2 above)
Comment (el-Joft) [reply .babelrc]: thanks.

Comment (el-Joft) [reply .babelrc]: done

**Review by pitaz — APPROVED** [review]

**Review by akpante3 — APPROVED** [review]

**Pull request merged**